README.md

```diff
@@ -93,6 +93,35 @@ find_package(Backward)
 # through an IMPORTED target.
 target_link_libraries(mytarget PUBLIC Backward::Backward)
 ```
+### Libraries to unwind the stack
```

Comment:
What about the section "Libraries to read the debug info"  Should not these 2 sections be merged ?

Reply:
I think the intent of the documentation is to describe the options you have to unwind the stack and, later, what you can do to read the debug info.

Reply:
Ok sorry my bad, it was not clear to me the difference between unwinding and reading debug since they look to unwind for me. Whatever ! Thank you for the clarification.